scalableminds/webknossos
# fix nml upload error message

### URL of deployed dev instance (used for testing):
- https://fixuploaderrormsg.webknossos.xyz

### Steps to test:
- upload NML for existing dataset, should work
- upload NML for non-existing dataset, should say that you don’t have access to it

### Issues:
- fixes #3576 

------
